### ISSUE TITLE
It keeps saying "Continue Server Starting".

### ISSUE THREAD

**Issue by dotyuu** [continuedev/continue#669]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Windows 11 
- Continue: v0.6.4
- python 3.12.1
- VsCode 1.85
```

### Description

Unable to connect to continue server，Even when I start it manually.

### To reproduce

For example, type /config
and then no response

### Log output

```Shell
[2023-12-12 19:09:23] [WARNING] Failed to load config from C:\Users\xiaoj\.continue\config.py: module 'config' has no attribute 'config'
[2023-12-12 19:09:23] [DEBUG] ------ Begin Logs ------
[2023-12-12 19:09:24] [DEBUG] Downloading MeiliSearch for Windows...
[2023-12-12 19:09:40] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:09:40] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:13:42] [INFO] There are 1 sessions currently open
[2023-12-12 19:14:25] [INFO] There are 1 sessions currently open
[2023-12-12 19:15:09] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:15:09] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:15:12] [WARNING] Error while building meilisearch index: MeilisearchApiError. Server error '503 Service Unavailable' for url 'http://localhost:7700/indexes'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503  
[2023-12-12 19:15:54] [INFO] There are 1 sessions currently open
[2023-12-12 19:15:55] [WARNING] Meilisearch did not start in less than 20 seconds. Stopping polling.
[2023-12-12 19:15:55] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:15:55] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:16:38] [INFO] There are 1 sessions currently open
[2023-12-12 19:16:41] [WARNING] Meilisearch did not start in less than 20 seconds. Stopping polling.
[2023-12-12 19:16:43] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:16:43] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:16:45] [WARNING] Error while building meilisearch index: MeilisearchApiError. Server error '503 Service Unavailable' for url 'http://localhost:7700/indexes'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503  
[2023-12-12 19:16:46] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:16:46] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:17:04] [WARNING] Meilisearch did not start in less than 20 seconds. Stopping polling.
[2023-12-12 19:17:07] [WARNING] Meilisearch did not start in less than 20 seconds. Stopping polling.
[2023-12-12 19:21:59] [INFO] There are 1 sessions currently open
[2023-12-12 19:22:04] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:22:04] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:22:05] [WARNING] Error while building meilisearch index: MeilisearchApiError. Server error '503 Service Unavailable' for url 'http://localhost:7700/indexes'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503  
[2023-12-12 19:22:25] [WARNING] Meilisearch did not start in less than 20 seconds. Stopping polling.
[2023-12-12 19:32:23] [WARNING] Error adding document to meilisearch: MeilisearchApiError.index_not_found Error message: Index `continue_context_items` not found. invalid_request Error documentation: https://docs.meilisearch.com/errors#index_not_found
[2023-12-12 19:32:27] [DEBUG] Health check
[2023-12-12 19:33:22] [INFO] There are 1 sessions currently open
[2023-12-12 19:34:04] [INFO] There are 1 sessions currently open
[2023-12-12 19:34:48] [INFO] There are 1 sessions currently open
[2023-12-12 19:34:49] [CRITICAL] Error handling message: Traceback (most recent call last):

  File "server\continuedev\server\gui.py", line 47, in message

  File "server\continuedev\server\protocols\gui.py", line 76, in handle_json

  File "server\continuedev\server\protocols\gui.py", line 183, in get_session_title

  File "server\continuedev\core\autopilot.py", line 344, in get_session_title

  File "server\continuedev\libs\llm\base.py", line 349, in complete

  File "server\continuedev\libs\llm\openai_free_trial.py", line 86, in _complete

  File "server\continuedev\libs\llm\proxy_server.py", line 39, in _complete

  File "aiohttp\client.py", line 1187, in __aenter__

  File "aiohttp\client.py", line 601, in _request

  File "aiohttp\client_reqrep.py", line 965, in start

  File "aiohttp\streams.py", line 622, in read

  File "asyncio\futures.py", line 287, in __await__

  File "asyncio\tasks.py", line 349, in __wakeup

  File "asyncio\futures.py", line 203, in result

aiohttp.client_exceptions.ServerDisconnectedError: Server disconnected

[2023-12-12 19:34:51] [INFO] Loaded Meilisearch index in 1.720 seconds
[2023-12-12 19:34:51] [INFO] Loaded Meilisearch index in 45.490 seconds
[2023-12-12 19:35:33] [DEBUG] Error loading meilisearch index: 
[2023-12-12 19:35:35] [INFO] Loaded Meilisearch index in 90.232 seconds
[2023-12-12 19:40:49] [DEBUG] Health check
[2023-12-12 19:41:39] [INFO] There are 1 sessions currently open
[2023-12-12 19:42:22] [INFO] There are 1 sessions currently open
[2023-12-12 19:42:26] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:42:26] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:42:28] [WARNING] Error while building meilisearch index: MeilisearchApiError. Server error '503 Service Unavailable' for url 'http://localhost:7700/indexes'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503  
[2023-12-12 19:43:10] [INFO] There are 1 sessions currently open
[2023-12-12 19:43:11] [WARNING] Meilisearch did not start in less than 20 seconds. Stopping polling.
[2023-12-12 19:43:11] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:43:11] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:43:54] [INFO] There are 1 sessions currently open
[2023-12-12 19:43:59] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:43:59] [INFO] Meilisearch started at http://localhost:7700
[2023-12-12 19:44:01] [WARNING] Error while building meilisearch index: MeilisearchApiError. Server error '503 Service Unavailable' for url 'http://localhost:7700/indexes'
For more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503  
[2023-12-12 19:44:01] [WARNING] Meilisearch did not start in less than 20 seconds. Stopping polling.
[2023-12-12 19:44:02] [DEBUG] Starting MeiliSearch...
[2023-12-12 19:44:02] [INFO] Meilisearch started at http://localhost:7700
```

**Comment by dotyuu** [comment]
When I try to start the server manually, it shows：
Traceback (most recent call last):
  File "urllib3\connectionpool.py", line 703, in urlopen
  File "urllib3\connectionpool.py", line 386, in _make_request
  File "urllib3\connectionpool.py", line 1042, in _validate_conn
  File "urllib3\connection.py", line 419, in connect
  File "urllib3\util\ssl_.py", line 449, in ssl_wrap_socket
  File "urllib3\util\ssl_.py", line 493, in _ssl_wrap_socket_impl
  File "ssl.py", line 517, in wrap_socket
  File "ssl.py", line 1108, in _create
  File "ssl.py", line 1383, in do_handshake
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "requests\adapters.py", line 486, in send
  File "urllib3\connectionpool.py", line 787, in urlopen
  File "urllib3\util\retry.py", line 592, in increment
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "litellm\__init__.py", line 61, in get_model_cost_map
  File "requests\api.py", line 73, in get
  File "requests\api.py", line 59, in request
  File "requests\sessions.py", line 589, in request
  File "requests\sessions.py", line 703, in send
  File "requests\adapters.py", line 517, in send
requests.exceptions.SSLError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /BerriAI/litellm/main/model_prices_and_context_window.json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "continue_server.py", line 4, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "server\main.py", line 1, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "server\continuedev\__init__.py", line 4, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "server\continuedev\core\config.py", line 10, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "server\continuedev\libs\llm\__init__.py", line 2, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "server\continuedev\libs\llm\ggml.py", line 9, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "server\continuedev\libs\llm\openai.py", line 6, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "litellm\__init__.py", line 71, in <module>
  File "litellm\__init__.py", line 68, in get_model_cost_map
  File "importlib\resources\_legacy.py", line 25, in wrapper
  File "importlib\resources\_legacy.py", line 63, in open_text
  File "pathlib.py", line 1044, in open
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\xiaoj\\AppData\\Local\\Temp\\_MEI133442\\litellm\\model_prices_and_context_window_backup.json'
[8752] Failed to execute script 'continue_server' due to unhandled exception!

**Comment by sestinj** [comment]
@dotyuu I finally have the long awaited solution. We've removed the need for the server, so the extension will just work. It's now available in pre-release, v0.7.32 on the VS Code extension marketplace.

If you see any more problems, I'd prefer to have them in a separate issue. Feel free to either open one or just comment here and I'll go ahead and create it
